src-web/nls/platform.properties

```diff
@@ -176,9 +176,10 @@ ansible.scheduleMode.once.info = When this policy is violated, the automation ru
 ansible.scheduleMode.disabled.title = Disable automation
 ansible.scheduleMode.disabled.info = Automation does not run until this setting is updated.
 ansible.unsaved.data = You made changes to the configuration. Do you want to save your policy automation? Click the Cancel button to close.
-ansible.loading.info = Loading policy violation automation data...
-ansible.create.success = New policy violation automation ${0} is successfully created on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
-ansible.edit.success = Existing policy violation automation ${0} is successfully updated on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
+ansible.loading.info = Loading policy violation automation data.
+ansible.manual.success = The automation has been manually initiated successfully. View the History tab to check the ansible history information.
+ansible.once.success = Policy automation is set to once mode. The automation is automatically initiated once when the policy is in a NonCompliant state. View the History tab to check the ansible history information.
+ansible.disabled.success = Suggestions mentioned recently should also be applied to this file if accepted.
```

Comment:
This was in my last comment, I think this should be something different right?

Reply:
@dockerymick 
<img width="743" alt="Screen Shot 2021-05-17 at 1 11 10 PM" src="https://user-images.githubusercontent.com/55596988/119560979-9cd41c00-bd72-11eb-9737-04912bdd5bcb.png">

Reply:
I suggest updating that to read:

The automation is not automatically initiated when it is set to disable mode.

Reply:
@ChunxiAlexLuo The suggestions weren't applied to the file. Can you avoid using the word "trigger". I recommend that we use initiate instead

Reply:
@dockerymick I think @ChunxiAlexLuo will fix this in his next pr.